src/toolbar_gui.cpp

```diff
@@ -669,60 +670,96 @@ static CallBackFunction MenuClickGoal(int index)
 	return CBF_NONE;
 }
 
-/* --- Graphs button menu --- */
-
-static CallBackFunction ToolbarGraphsClick(Window *w)
-{
-	PopupMainToolbMenu(w, WID_TN_GRAPHS, STR_GRAPH_MENU_OPERATING_PROFIT_GRAPH, (_toolbar_mode == TB_NORMAL) ? 6 : 8);
-	return CBF_NONE;
-}
+/* --- Graphs and League Table button menu --- */
 
 /**
- * Handle click on the entry in the Graphs menu.
- *
- * @param index Graph to show.
- * @return #CBF_NONE
+ * Enum for the League Toolbar's and Graph Toolbar's related buttons.
```

Comment:
Not actually an enum. Could it be?

Reply:
Possibly, but `CTMN_*` in the same file isn't technically a enum either but is called so in the comment.